tests/util/setup-smoke-tests.ts

```diff
@@ -1,16 +1,15 @@
-import { ApiPromise, WsProvider } from "@polkadot/api";
+import { ApiPromise } from "@polkadot/api";
 import { MockProvider } from "@polkadot/rpc-provider/mock";
 import { TypeRegistry } from "@polkadot/types";
-import { types } from "moonbeam-types-bundle";
-import { ethers } from "ethers";
+import { providers } from "ethers";
+import { SubstrateApi, EthersApi, ApiType } from "./wsApis";
 
 const debug = require("debug")("test:setup");
 
 export interface SmokeTestContext {
-  // We also provided singleton providers for simplicity
   polkadotApi: ApiPromise;
   relayApi: ApiPromise;
-  ethers?: ethers.providers.WebSocketProvider;
+  ethers: providers.WebSocketProvider;
 }
 
 export type SmokeTestOptions = {
```

Comment:
This should disappear now that we use the env in the global setup

Reply:
We should keep these here as the SmokeTestContext provides a common way for all tests to get their providers.

Reply:
Oh i see what you mean. Refactoring now.